source/extensions/tracers/xray/BUILD

```diff
@@ -59,7 +59,6 @@ envoy_cc_extension(
     hdrs = ["config.h"],
     category = "envoy.tracers",
     security_posture = "robust_to_untrusted_downstream",
-    status = "wip",
```

Comment:
I'm not sure we can bump the AWS extensions from `wip` or `alpha` and claim security until they move away from libcurl. https://github.com/envoyproxy/envoy/issues/11816. We have a very high CVE rate in this library, probably an order-of-magnitude higher than any other Envoy dep.

Reply:
Hi @htuch, thank you for taking a look at this PR and for the comment. This doc specifically remove the "wip" for the AWS X-Ray tracing extension [API](https://storage.googleapis.com/envoy-pr/276708a/docs/api-v3/config/trace/v3/xray.proto.html) which doesn't use the libcurl module or curl utility. There are other AWS extension that use it as per mentioned here at [external_deps](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/external_deps.html?highlight=curl). And the AWS X-Ray doesn't depend on these other AWS extensions as well.

Reply:
Referring to the [comment](https://github.com/envoyproxy/envoy/pull/8906#issuecomment-552538354) left by the original author of this AWS X-Ray extension on an old #8906 PR.

Reply:
I concur that this extension doesn't use curl. @htuch are ok with this? Seems fine to no longer be WIP.

Reply:
Objection overruled  :)